docs/src/modules/utils/generateMarkdown.ts

```diff
@@ -476,15 +476,22 @@ function generateClasses(reactAPI: ReactApi) {
 ${text}
 
 You can override the style of the component thanks to one of these customization points:
-
+${
+  styledComponent
+    ? `
+- With a [global class name](/guides/interoperability/#global-css).
+- With a rule name as part of the theme's component's [\`styleOverrides\` property](/customization/components/#global-theme-override).
```

Comment:
Is there another way of wording that? It's a bit hard to parse.

```suggestion
- With a rule name as part of the component's [\`styleOverrides\` property](/customization/components/#global-theme-override) in a custom theme.
```
?